teos/src/main.rs

```diff
@@ -244,9 +246,25 @@ async fn main() {
         .parse()
         .unwrap();
 
+    // Generate mtls certificates to data directory so the admin can securely connect
+    // to the server to perform administrative tasks.
+    let (identity, ca_cert) = tls_init(&path).unwrap_or_else(|e| {
+        eprintln!("Couldn't generate tls certificates: {:?}", e);
+        std::process::exit(1);
+    });
+    let identity = identity.to_tonic_identity();
+    let ca_cert = tonic::transport::Certificate::from_pem(ca_cert);
+
+    let tls = tonic::transport::ServerTlsConfig::new()
```

Comment:
nit: import `tonic:transport:{Certificate, ServerTlsConfig}`

Reply:
nit: Also, this can all be made into a single call without the intermediate vars